api/db-cdr/changelog-data/db.changelog-7.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.0.xsd
+        http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd">
+  <changeSet author="biopete" id="changelog-7" runAlways="true" context="local or cloud">
```

Comment:
Do we need to specify "local or cloud" for these? If you didn't specify any context, would it run in all contexts?

Reply:
this is specifically for the icd10 tree which has close to 300,000 rows. local will load a subset and cloud will load the real tree.

Reply:
What I meant is, if you don't specify context here (since we want it to run for local or cloud), will it run when local or cloud context are provided on the command line?

Reply:
I will test and see what happens if i remove the context.

Reply:
i removed it from changelog-7 and it doesn't get run when running: ../gradlew update -PrunList=data -Pcontexts=local